core/services/fluxmonitorv2/poll_manager.go

```diff
@@ -138,6 +138,11 @@ func (pm *PollManager) Start(hibernate bool, roundState flux_aggregator_wrapper.
 		}()
 	}
 
+	if !pm.cfg.IdleTimerDisabled && !pm.cfg.PollTickerDisabled && pm.cfg.IdleTimerPeriod < pm.cfg.PollTickerInterval {
```

Comment:
Consider extracting this out into a function on the poll manager for ease of understanding.

Reply:
👍🏼